src/Controllers/WorkItemController.php

```
@@ -747,20 +747,12 @@ public function refineAll(): void
             return;
         }
 
-        $rows = $this->db->query("SELECT wi.id FROM hl_work_items wi
-             JOIN projects p ON p.id = wi.project_id
-             WHERE wi.project_id = :pid
-               AND p.org_id = :oid
-               AND wi.quality_status = 'scored'
-               AND wi.quality_score < 80
-             ORDER BY wi.quality_score ASC
-             LIMIT 50", [':pid' => $projectId, ':oid' => $orgId])->fetchAll();
+        $itemIds = HLWorkItem::lowQualityIdsForProjectAndOrg($this->db, $projectId, $orgId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\HLWorkItem' in method 'refineAll'. (undefined)

(StaticAccess)